com.unity.cinemachine/Editor/Utility/InspectorUtility.cs

```diff
@@ -519,6 +519,50 @@ public static void AddPropertyDragger(this Label label, SerializedProperty p, Vi
             }
         }
         
+        /// <summary>A small warning sybmol, suitable for embedding in an inspector row</summary>
+        /// <param name="tooltip">The tooltip text</param>
+        public static Label MiniHelpIcon(string tooltip, HelpBoxMessageType iconType = HelpBoxMessageType.Warning)
+        {
+            string icon = iconType switch
+            {
+                HelpBoxMessageType.Warning => "console.warnicon.sml",
+                HelpBoxMessageType.Error => "console.erroricon.sml",
+                _ => "console.infoicon.sml",
+            };
+            return new Label 
+            { 
+                tooltip = tooltip,
+                style = 
+                { 
+                    flexGrow = 0,
+                    backgroundImage = (StyleBackground)EditorGUIUtility.IconContent(icon).image,
+                    width = SingleLineHeight, height = SingleLineHeight,
+                    alignSelf = Align.Center
+                }
+            };
+        }
+
+        /// <summary>A small popup context menu, suitable for embedding in an inspector row</summary>
+        /// <param name="tooltip">The tooltip text</param>
+        /// <param name="contextMenu">The context menu to show when the button is pressed</param>
+        public static Button MiniPopupButton(string tooltip = null, ContextualMenuManipulator contextMenu = null)
+        {
+            var button = new Button { tooltip = tooltip, style = 
+            {
+                backgroundImage = (StyleBackground)EditorGUIUtility.IconContent("_Popup").image,
+                width = InspectorUtility.SingleLineHeight, height = InspectorUtility.SingleLineHeight,
+                alignSelf = Align.Center,
+                paddingRight = 0, borderRightWidth = 0, marginRight = 0
```

Comment:
Each param should be on a new line in my opinion.
There is no need for `InspectorUtility.`.